.env

```
@@ -350,6 +350,8 @@ VITE_TENDERLY_PROJECT_SLUG=project
 # Webservices
 VITE_SWAPS_SERVER_URL=https://api.swap-service.shapeshift.com
 VITE_USER_SERVER_URL=https://api.user-service.shapeshift.com
+# TODO(placeholder): set to the real public-api prod hostname. Browser resolves partner codes via public-api now.
+VITE_PUBLIC_API_URL=https://api.shapeshift.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_PUBLIC_API_URL key should go before the VITE_SWAPS_SERVER_URL key

(UnorderedKey)